### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by mounor** [SDRausty/TermuxArch#288] [General]
hello

**Comment by SDRausty** [comment]
hello world

**Reply by mounor** [reply]
# good morning 
#can i get your help ? 🙂

**Comment by SDRausty** [comment]
# good morning 
#with what? 🎼

**Reply by mounor** [reply]
in language termux i want to learn it if you can help me 🙂❤

**Reply by SDRausty** [reply]
Does this link https://tldp.org/guides.html help?

**Reply by mounor** [reply]
[ ] thank you

**Reply by SDRausty** [reply]
☑️ you're welcome

**Comment by 0xGodspeed** [comment]
hi

**Reply by SDRausty** [reply]
hello

**Reply by 0xGodspeed** [reply]
how long does the unpacking take during the installation?

**Comment by SDRausty** [comment]
>how long does the unpacking take during the installation?

Not that long at all, if your Internet connection is 64k unpacking and configuration is much shorter than the download time.

**Comment by 0xGodspeed** [comment]
Are there no modules? I want the tun module because openvpn requires it
